.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "cloudendure_demo/playbooks/roles/gogs"]
```

Comment:
Josh,

Thanks I went through the licenses with legal and they are all MIT/BSD and it was approved.